rust/src/linkspsql.rs

```diff
@@ -0,0 +1,141 @@
+extern crate tokio_postgres;
+extern crate platform_data;
+extern crate platform_num;
+
+use std::borrow::Borrow;
+use std::ops::{AddAssign, Deref};
+use platform_num::LinkType;
+use platform_data::LinksConstants;
+use tokio::runtime::Runtime;
+
+pub use tokio_postgres::{Client, NoTls, tls::NoTlsStream, Error, Row};
+use tokio_postgres::types::FromSql;
+
+pub struct LinksPSQL {
+    pub index: u32,
+    pub client: Client,
+}
+impl LinksPSQL
+{
+    pub fn new(client: Client) -> Result<Self, Error> {
+        let run = Runtime::new().expect("Something went wrong with tokio::runtime");
+        let index = run.block_on(
+            client.query("SELECT * FROM Links", &[])
+        ).expect("");

```

Comment:
But delegate error from this function
```suggestion
        ).unwrap();

```